app/views/rules/index.html.erb

```diff
@@ -33,15 +33,41 @@
 <div class="bg-container shadow-border-xs rounded-xl p-4">
 
   <% if @rules.any? %>
-    <div class="rounded-xl bg-gray-25 space-y-1">
-      <div class="flex items-center gap-1.5 px-4 py-2 text-xs font-medium text-secondary uppercase">
-        <p>Rules</p>
-        <span class="text-subdued">&middot;</span>
-        <p><%= @rules.count %></p>
+    <div class="rounded-xl bg-gray-25">
+      <div class="flex items-center justify-between px-4 py-2 text-xs uppercase">
+        <div class="flex items-center gap-1.5 font-medium text-secondary">
+          <p>Rules</p>
+          <span class="text-subdued">&middot;</span>
+          <p><%= @rules.count %></p>
+        </div>
+        
+        <div class="flex items-center gap-1">
+          <span class="text-secondary">Sort by:</span>
+          <div class="flex items-center">
+            <%= link_to rules_path(sort_by: "name", direction: (@sort_by == "name" && @direction == "asc") ? "desc" : "asc"), 
+                  class: "px-2 py-1 rounded flex items-center hover:bg-gray-100 #{@sort_by == 'name' ? 'font-medium' : ''}" do %>
+              <span>Name</span>
+              <% if @sort_by == "name" %>
+                <%= lucide_icon(@direction == "asc" ? "arrow-up" : "arrow-down", class: "w-3 h-3 text-accent ml-1") %>
+              <% end %>
+            <% end %>
+          </div>
+          <div class="flex items-center">
+            <%= link_to rules_path(sort_by: "created_at", direction: (@sort_by == "created_at" && @direction == "asc") ? "desc" : "asc"), 
+                  class: "px-2 py-1 rounded flex items-center hover:bg-gray-100 #{@sort_by == 'created_at' ? 'font-medium' : ''}" do %>
+              <span>Date</span>
+              <% if @sort_by == "created_at" %>
+                <%= lucide_icon(@direction == "asc" ? "arrow-up" : "arrow-down", class: "w-3 h-3 text-accent ml-1") %>
+              <% end %>
+            <% end %>
+          </div>
+        </div>
       </div>
 
       <div class="space-y-1 p-1">
-        <%= render @rules %>
+        <% @rules.each do |rule| %>
+          <%= render "rule", rule: rule %>
+        <% end %>
```

Comment:
For some reason brakeman started warning about these lines so instead of a dynamic render I'm doing it a bit more explicitly. This isn't something I changed directly, although maybe the sorting changed how this works here. I don't quite understand this - I'd love some more insight here from someone who more ruby/rails than I do.

```
Dynamic Render Path: 1

== Warnings ==

Confidence: Weak
Category: Dynamic Render Path
Check: Render
Message: Render path contains parameter value
Code: render(action => Current.family.rules.order(validate_sort_param(params[:sort_by], "name") => validate_direction_param(params[:direction], "asc")), {})
File: app/views/rules/index.html.erb
Line: 68
```